CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ Changelog for NeoFS Node
 ### Fixed
 
 ### Removed
-
+- Remove WIF and NEP2 support in --wallet argument
```

Comment:
the last edit:

```suggestion
- WIF and NEP2 support in `neofs-cli`'s --wallet flag (#1128)
```

- we specify an issue number if exists or a current PR number if we work without an issue
- that repo contains 5 (and could more in the future) executables so it could be helpful to mention what exactly binary has changed its behavior

Reply:
Thanks, addressed.